File: IntegrationGuideForAndroid/Apps/trafficAuthentication/src/main/java/com/example/integrationguide/retrofit/RetrofitAPIInterface.java
// Copyright 2018 Omnissa, LLC.
// SPDX-License-Identifier: BSD-2-Clause

package com.example.integrationguide.retrofit;

import retrofit2.Call;
import retrofit2.http.GET;

public interface RetrofitAPIInterface {

    @GET("/api/users/2")
    Call<User> getUserInfo();
}

File: IntegrationGuideForAndroid/Apps/tunnel/src/main/java/com/example/integrationguide/retrofit/RetrofitAPIInterface.kt
// Copyright 2023 VMware, Inc.
// SPDX-License-Identifier: BSD-2-Clause

package com.example.integrationguide.retrofit

import retrofit2.Call
import retrofit2.http.GET

interface RetrofitAPIInterface {
    @get:GET("/api/users/2")
    val userInfo: Call<User?>?
}